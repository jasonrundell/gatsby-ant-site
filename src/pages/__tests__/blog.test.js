import React from 'react'
import renderer from 'react-test-renderer'

import Blog from '../blog'

describe('Blog', () => {
  it('renders correctly', () => {
    const data = {
      site: {
        siteMetadata: {
          title: '',
        },
      },
      allMarkdownRemark: {
        edges: [
          {
            node: {
              excerpt: '',
              id: '',
              frontmatter: {
                title: '',
                date: '',
                path: '',
                featuredImage: '',
                featuredImageAlt: '',
              },
            },
          },
        ],
      },
    }

    const tree = renderer.create(<Blog data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
