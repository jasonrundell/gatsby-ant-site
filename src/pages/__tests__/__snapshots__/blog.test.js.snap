// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog renders correctly 1`] = `
Array [
  <a
    className="link"
    href="#main"
  >
    Skip to main content
  </a>,
  <header
    className="header"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        <a
          className="link"
          href="/"
        >
          
        </a>
      </h1>
      <div
        className="searchContainer"
      >
        <a
          className="openSearch"
          href="/search"
          onClick={[Function]}
        >
          <svg
            className="icon"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Search the blog
            </title>
            <path
              d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
            />
          </svg>
        </a>
        <div
          className="overlay"
        >
          <button
            onClick={[Function]}
          >
            close
          </button>
        </div>
      </div>
    </div>
  </header>,
  <div
    className="container"
  >
    <nav
      aria-label="Breadcrumb"
      className="breadcrumb"
    >
      <ol>
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/blog/"
          >
            Blog
          </a>
        </li>
      </ol>
    </nav>
    <main
      id="main"
    >
      <ul
        className="list"
      />
    </main>
    <footer
      className="footer"
    >
      Â© 
      2019
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        rel="noopener noreferrer"
      >
        Gatsby
      </a>
      .
       
      Calin says hello.
    </footer>
  </div>,
]
`;
